CHANGELOG.md

```diff
@@ -11,6 +11,24 @@ Semantic Versioning is maintained only for the following:
 The fixers themselves can change their behavior on any update.
 New fixers could be added with minor releases, this would require changes in configuration if migration mode is used.
 
+## 3.0.1
+
+### Changed
+- Minimum PHP version has been changed to 8.1.
+
```

Comment:
empty lines not needed in changelog list

Reply:
fixed